client/galaxy/scripts/nls/fr/locale.js

```diff
@@ -6,61 +6,61 @@ define({
 "Workflow":
     "Workflow",
 "Shared Data":
-    "Partage de données",
+    "Données partagées",
 "Data Libraries":
-    false,
+    "Bibliothèque de données",
 "Histories":
-    false,
+    "Historiques",
 "Workflows":
-    false,
+    "Workflows",
 "Visualizations":
-    false,
+    "Visualisations",
 "Pages":
-    false,
+    "Pages",
 "Visualization":
-    "Représentation des données",
+    "Visualisation",
 "New Track Browser":
-    false,
+    "Nouveau Navigateur de Tracks/Pistes",
```

Comment:
Not easy to translate... Maybe something like "Nouvelle visualisation de génome" or as it is through Trackster something like "Nouvelle visualisation Trackster"...